src/Makefile.OCaml

```diff
@@ -72,7 +72,14 @@ endif
 
 .PHONY: macuimaybe
 ifeq ($(OSARCH), Darwin)
-  macuimaybe: macui
+  # If XCode is not installed, xcodebuild is just a placeholder telling that XCode is not installed
+  # and any invocation of xcodebuild results in a non 0 exit code.
+  ifeq ($(shell xcodebuild -version > /dev/null; echo $$?), 0)
+    macuimaybe: macui
+  else
+    macuimaybe:
+      $(info Not building macOS native GUI because XCode is not installed.)
```

Comment:
While I don't think it is required here, for consistency with other recipes, I propose to replace spaces with a tab here:
```suggestion
	$(info Not building macOS native GUI because XCode is not installed.)
```

Reply:
Nevermind, I get it now.   I withdraw the question about style guide.

@EruEri The issue is that in makefiles, the commands to implement a target are indented with a tab character.  The rest of the file does it that way.

Reply:
No, and probably not. I only propose a tab here because regular recipes require a tab, so that's how all recipes have been written. Even those that are not 'regular'.

Reply:
Maybe we can indent target with spaces and indent recipes with the same amout of space + the required tabulation ?

Reply:
If so, the spaces would have to be after, and I think that's messier.  I called it good enough as I think we've hit diminishing returns....